ubyssey/events/models.py

```diff
@@ -99,3 +99,8 @@ def send_published_email(sender, instance, **kwargs):
 
         instance.is_published_email = True
         instance.save()
+
+class ScrapedEvent(Model):
+
+    GUID = CharField(max_length=50)
```

Comment:
I would set `primary_key=True` for this field

Reply:
Also, I think it would be better to use lowercase `guid` everywhere instead of all caps